client/watch.html

```diff
@@ -666,6 +665,11 @@
               <center>
                 <button class="curator-card-like favorite-button {{#if userFavorited}}unfavorite{{else}}favorite{{/if}}">{{> favorite_icon}} Show some love</button>
               </center>
+              {{#if replayAvailable}}
+              <center>
```

Comment:
yes, I have a note to amend these. As we are doing a re-layout of the watch page currently its in my notes to make sure I don't reintroduce my love of `<center>` tags 👍